airy.go

```diff
@@ -0,0 +1,29 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import "github.com/gonum/mathext/internal/amos"
+
+// Airy returns the Airy function at z. The Airy function here, Ai(y), is one
```

Comment:
According to the link, all of the solutions are Airy functions, so I think it is necessary to specify that it is Ai(z)